apps/nextra/pages/en/build/cli/trying-things-on-chain/ledger.mdx

```diff
@@ -117,3 +119,495 @@ yes
+</Callout>
+
+<Steps>
+
+### Complete the key rotation guide
+
+Confirm that you have completed the
+[key rotation guide](../../advanced-guides/key-rotation.mdx).
+
+### Verify your Ledger is ready
+
+1. Connect and unlock your Ledger.
+1. Check what version of the Aptos app you have: `Aptos > About > Version`.
+1. If you do not have version `0.6.9` or higher, update it using Ledger Live.
+1. Enable blind signing: `Aptos > Settings > Enable Blind Signing`.
+
+### Start a localnet
+
+Start a localnet:
+
+```sh filename="Terminal"
+aptos node run-localnet
+```
+
+The localnet is ready when it prints out:
+
+```sh filename="Terminal"
+Applying post startup steps...
+
+Setup is complete, you can now use the localnet!
+```
+
+<Callout type="info" emoji="🧠">
+If you are on a UNIX-like system, the following command can be used to start a
+fresh localnet as a background process:
+
+```sh filename="Terminal"
+mkdir -p localnet-data
+aptos node run-localnet \
+    --assume-yes \
+    --test-dir localnet-data \
+    --force-restart &
+export LOCALNET_PID=$!
+```
+
+You can then stop the localnet at any point with the following command:
+
+```sh filename="Terminal"
+kill $LOCALNET_PID
+```
+</Callout>
+
+### Set up localnet hot wallet profile
+
+Create a private key corresponding to an authentication key, and thus initial
+account address, that starts with the vanity prefix `0xaaa`:
+
+```sh filename="Terminal"
+aptos key generate \
+    --assume-yes \
+    --output-file private-key-a \
+    --vanity-prefix 0xaaa
+```
+
+<details>
+<summary>Example output</summary>
+```sh filename="Terminal"
+{
+  "Result": {
+    "PublicKey Path": "private-key-a.pub",
+    "PrivateKey Path": "private-key-a",
+    "Account Address:": "0xaaac71af5f2a4af4ec2639a15799bf9b945afb061c8bee102b636531c1b00eb5"
+  }
+}
+```
+</details>
+
+Use the private key to initialize a `hot-wallet-1` profile on the localnet:
+
+```sh filename="Terminal"
+aptos init \
+    --assume-yes \
+    --network local \
+    --private-key-file private-key-a \
+    --profile hot-wallet-1
+```
+
+<details>
+<summary>Example output</summary>
+```sh filename="Terminal"
+Configuring for profile hot-wallet-1
+Configuring for network Local
+Using command line argument for private key
+Account 0xaaac71af5f2a4af4ec2639a15799bf9b945afb061c8bee102b636531c1b00eb5 doesn\'t exist, creating it and funding it with 100000000 Octas
+Account 0xaaac71af5f2a4af4ec2639a15799bf9b945afb061c8bee102b636531c1b00eb5 funded successfully
+
+---
+Aptos CLI is now set up for account 0xaaac71af5f2a4af4ec2639a15799bf9b945afb061c8bee102b636531c1b00eb5 as profile hot-wallet-1!  Run `aptos --help` for more information about commands
+{
+  "Result": "Success"
+}
+```
+</details>
+
+### Rotate the hot wallet key
+
+Rotate the authentication key of the hot wallet to use [BIP44 account index]
+1000 on your Ledger:
+
+```sh filename="Terminal"
+aptos account rotate-key \
+    --assume-yes \
+    --new-derivation-index 1000 \
```

Comment:
As I mentioned in the PR description

> Install the CLI from source, as described in https://github.com/aptos-labs/aptos-core/pull/11151

This PR pairs simply because I *had* to split docs and source into two separate repos https://github.com/aptos-labs/aptos-core/pull/11151

Reply:
Noting that I've updated the PR description per the recent set of `aptos-core` PRs that have merged, to stipulate installing the latest Framework updates